pkg/operator/observe_config.go

```diff
@@ -105,6 +111,20 @@ func (c ConfigObserver) sync() error {
 	return nil
 }
 
+func observeCABundleConfig(kubeClient kubernetes.Interface, clientConfig *rest.Config, observedConfig map[string]interface{}) (map[string]interface{}, error) {
+	caBundleConfig, err := kubeClient.CoreV1().ConfigMaps(operatorNamespaceName).Get("service-ca-bundle-configmap", metav1.GetOptions{})
+	if err != nil && !errors.IsNotFound(err) {
+		return nil, err
+	}
+	if caBundleConfig != nil {
+		// TODO(juanvallejo): reflect any issues in operator status
+		if val := caBundleConfig.Data["service-ca.crt"]; len(val) > 0 {
+			// TODO: what is the nested field path? Where do we store the service-ca info?
```

Comment:
@deads2k or @sttts are we adding this to a volume definition on the operator config?

Reply:
> @deads2k or @sttts are we adding this to a volume definition on the operator config? Not sure of the path here

You actually need to do two things: 
1) do a `SyncConfigMap` call in the main config to copy the CA locally
2) watch for that change and if you have a value, set this value for a path.  You need to create a new bit in our binary's config (kubecontorlplane group)